cypress/integration/visitorCanSeeMultipleLanguages.spec.js

```diff
@@ -0,0 +1,25 @@
+describe("Visitor can change language from", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v0/articles",
+      response: "fixture:articles.json"
+    });
+    cy.visit("http://localhost:3001");
+    cy.get('#english').click({ force: true });
+  });
+
+  it("English to Swedish", () => {
+      cy.get('#homebutton').should('contain', 'Home');
```

Comment:
The convention dictates that we use so-called kebab case on class names and id's For this, it would mean: `home-button`. The reason for this is readability.

Reply:
Fixed.